$NetBSD: patch-aa,v 1.4 2014/03/14 20:49:56 gdt Exp $

Disable installation of config file, because pkgsrc installs config
files as examples.

--- etc/Makefile.am.orig	2012-12-04 14:43:40.000000000 +0000
+++ etc/Makefile.am
@@ -22,11 +22,3 @@ opensc.conf: opensc.conf.in force
 		-e 's|@DEFAULT_SM_MODULE[@]|$(DEFAULT_SM_MODULE)|g' \
 		-e 's|@DEBUG_FILE[@]|$(DEBUG_FILE)|g' \
 		< $< > $@
-
-install-exec-hook: opensc.conf
-	$(MKDIR_P) "$(DESTDIR)$(sysconfdir)"
-	if [ -f "$(DESTDIR)$(sysconfdir)/opensc.conf" ]; then \
-		$(INSTALL_DATA) opensc.conf "$(DESTDIR)$(sysconfdir)/opensc.conf.new"; \
-	else \
-		$(INSTALL_DATA) opensc.conf "$(DESTDIR)$(sysconfdir)/opensc.conf"; \
-	fi
